sauce/features/accounts/accounts-row-height/index.js

```diff
@@ -0,0 +1,55 @@
+import Feature from 'core/feature';
+
+const compactHeight = 27;
+const slimHeight = 22;
+
+export default class AccountsRowHeight extends Feature {
+  injectCSS() {
+    let css = require('./index.css');
+
+    if (this.settings.enabled === '1') {
+      css += require('./compact.css');
+    } else if (this.settings.enabled === '2') {
+      css += require('./slim.css');
+    }
+
+    return css;
+  }
+
+  invoke() {
+    // If the activity-transaction-link feature is not active we don't want to adjust the
+    // record height because doing so causes scrolling to be jumpier than usual. If the
+    // activity-transaction-link feature is active we do need to set the record height
+    // because it uses the value to scroll the "selected transaction" to the top of the
+    // register.
+    if (ynabToolKit.options.activityTransactionLink) {
+      let ynabGridContainer = ynabToolKit.shared.getEmberView($('.ynab-grid-container').attr('id'));
```

Comment:
You can bring in the helper here and use the getEmberView on it instead of using the shared object